File: tests/indexing/parsing/lang/fixtures/java/Conditionals.java
class Conditionals {

    String describe(int score) {
        String tier;
        if (score >= 90) {
            tier = "gold";
            score = score - 90;
        } else if (score >= 50) {
            tier = "silver";
        } else {
            tier = "bronze";
            score = 0;
        }
        return tier + ":" + score;
    }

    int settle(int value, int min, int max) {
        int adjusted = value;
        if (value < min) {
            int deficit = min - value;
            adjusted = min + deficit / 2;
        }
        if (value > max) {
            adjusted = max;
            return adjusted - 1;
        }
        return adjusted;
    }
}

File: tests/indexing/parsing/lang/fixtures/kotlin/Conditionals.kt
class Conditionals {

    fun describe(score: Int): String {
        var tier: String
        if (score >= 90) {
            tier = "gold"
            return tier + score
        } else if (score >= 50) {
            tier = "silver"
        } else {
            tier = "bronze"
            tier = tier + "!"
        }
        return tier
    }

    fun settle(value: Int, min: Int, max: Int): Int {
        var adjusted = value
        if (value < min) {
            val deficit = min - value
            adjusted = min + deficit / 2
        }
        if (value > max) {
            adjusted = max
            return adjusted - 1
        }
        return adjusted
    }
}
